lib/stat_tracker.rb

```diff
@@ -1,3 +1,10 @@
+require_relative 'game'
+require_relative 'team'
+require_relative 'game_team'
+require_relative 'season_stats'
+require_relative 'league_stats'
+require_relative 'team_stats'
```

Comment:
great! I kept getting errors running the spec harness and I think this fixes it. nice catch.